parametric_stellarator.py

```diff
@@ -19,15 +22,24 @@ def cubit_export(solids, h5m_tags, facet_tol, len_tol, norm_tol):
         len_tol (float): maximum length of facet edge for Cubit export.
         norm_tol (float): maximum change in angle between normal vector of
             adjacent facets.
+        include_magnets (bool): generate magnets based on user-supplied coil
+            data.
+        magnet_h5m_tag (str): string to use for material tag in H5M model.
     """
     # Get current working directory
     cwd = os.getcwd()
 
     # Initialize import command
     impt_cmd = 'import step "' + cwd + '/'
 
-    # Initialize Cubit
-    cubit.init([''])
+    # If not already initialized by generating magnets, initialize Cubit and
+    # volume index
+    if include_magnets == False:
+        cubit.init([''])
+        vol_id = 1
+    # If magnets generated, retrieve index of last volume created
+    else:
+        vol_id = cubit.get_last_id("volume")
```

Comment:
In this version of the code, the H5M tag for the magnets is input via the `magnets` dictionary, and this H5M tag is not used in the `magnet_coils` script. I can have `magnet_h5m_tag` passed separately though so that the arguments don't need to be parsed before calling `magnet_coils`